<?xml version="1.0"?>
<launch>

 <!-- small description about your node -->
 
 <node pkg="mission_planner" type="mission_planner_node" name="MissionPlannerNode" respawn="false" output="screen">
	<rosparam command="load" file="$(find mission_planner)/config/config_mission_planner.yaml"/>
</node>

</launch>
